myExpenses/src/main/java/org/totschnig/myexpenses/provider/BackupUtils.kt

```diff
@@ -79,7 +79,7 @@ private fun sync(contentResolver: ContentResolver, backend: String?, backupFile:
         var backupFileName = backupFile.name
         if (backupFileName == null) {
             CrashHandler.report(Exception("Could not get name from uri ${backupFile.uri}"))
-            backupFileName = "backup-" + SimpleDateFormat("yyyMMdd", Locale.US)
+            backupFileName = "MyExpenses-backup-" + SimpleDateFormat("yyyMMdd", Locale.US)
```

Comment:
Actually this is not where the file for the normal backup operation is built, this is a fallback if after saving a backup, retrieving the name from the result fails.